ui/urls.py

```diff
@@ -20,4 +24,9 @@
     url(r"^terms_and_conditions$",
         cache_me(CachableTemplateView.as_view(template_name="terms.html")),
         name="terms"),
+
+    url(r"^step1$",
```

Comment:
Form for a single company can be submitted just once, so no need for sharable url.

Please use https://django-formtools.readthedocs.io/en/latest/ for the form wizard,